airflow/contrib/operators/sqoop_operator.py

```diff
@@ -36,7 +43,7 @@ def __init__(self,
                  query=None,
                  target_dir=None,
                  append=None,
-                 file_type=None,
+                 file_type='text',
```

Comment:
if we don't default this to 'text', it breaks. comments say it defaults to 'text' but it doesn't because in the sqoop_hook when you try to assign the `file_type` to None it just returns an exception in `_get_export_format_argument`. Also this is clearer

Reply:
Makes sense